client/src/components/CreatePost.tsx

```diff
@@ -37,14 +53,31 @@ function CreatePost() {
     setForm({ ...form, file: url });
   };
 
+const [isOpen, toggleDialog] = React.useState(false);
+  const closeDialog = () => {
+    toggleDialog(false);
+  };
+
+
   return (
+    <>
+      <Button
+        variant="outlined"
+        onClick={() => {
+          toggleDialog(true);
+        }}
+        data-testid="btn-newpost"
+      >
+        New Post
```

Comment:
`Create a Post`?

Reply:
I'll actually revert this file to the state of GDSC main since Gurvir has this in his PR